sdk/monitor/monitor-ingestion/src/models.ts

```diff
@@ -37,12 +35,28 @@ export interface UploadLogsError {
 }
 
 /**
- * Type representing whether all or few logs succeeded uploading
+ * Aggregate Upload Logs Error Name
  */
-export type UploadLogsStatus =
-  /** Represents Complete Failure scenario where all logs have failed for processing and the list of logs that failed to upload are returned */
-  | "Failure"
-  /** Represents Partial Failure scenario where partial logs have failed for processing and the list of logs that failed to upload are returned */
-  | "PartialFailure"
-  /** Represents Success scenario where all logs have succeeded and no index is returned */
-  | "Success";
+export const AggregateUploadLogsErrorName = "AggregateUploadLogsError";
+
+/**
+ * Aggregate Error type for upload function
+ */
+export class AggregateUploadLogsError extends Error {
```

Comment:
can we export a `isAggregateUploadLogsError` typeguard for this and use it in the samples?

Reply:
done